src/index.js

```diff
@@ -50,13 +50,15 @@ function updateV8HeapSpaceGauges(registry, extraTags, heapSpaceStats) {
 }
 
 class RuntimeMetrics {
-  constructor(registry) {
+  constructor(registry, opts) {
     if (typeof process.cpuUsage !== 'function' ||
       typeof v8.getHeapSpaceStatistics !== 'function') {
       throw new Error('nflx-spectator-nodemetrics requires node.js 6 or newer');
     }
 
     this.registry = registry;
+    // delay initial cpu measurement for 30 seconds (by default) so that the first value isn't artificially high
+    this.cpuGaugeStartDelay = opts && Number.isInteger(opts.cpuGaugeStartDelay) ? opts.cpuGaugeStartDelay : 30000;
```

Comment:
Do we actually want to default this to zero in case other non-NQ services are using this lib and depending on the current behavior for some reason?